Firmware/Marlin_main.cpp

```diff
@@ -76,7 +76,6 @@
 #include "cardreader.h"
 #include "ConfigurationStore.h"
 #include "language.h"
-#include "pins_arduino.h"
```

Comment:
this file does in fact exist. It's part of the arduino board variant (Einsy rambo in this case). See https://github.com/prusa3d/Arduino_Boards/blob/master/IDE_Board_Manager/prusa3dboards/variants/prusa_einsy_rambo/pins_arduino.h

Reply:
I see it is included with `Arduino.h` in `Marlin.h` so I believe we are safe in removing this particular include in `Marlin_main.cpp`

I'll reword the commit message "doesn't exist" is inaccurate as you point out 👍 

![image](https://user-images.githubusercontent.com/8218499/209663926-8b408fce-8f2f-443b-8a4a-a6fe2054e0b0.png)